ocs_ci/helpers/dr_helpers.py

```diff
@@ -560,6 +579,7 @@ def wait_for_replication_resources_creation(
         timeout (int): time in seconds to wait for VR or ReplicationSource resources to be created
             or reach expected state
         discovered_apps (bool): If true then deployed workload is discovered_apps
+        vrg_name (str): Name of VRG
```

Comment:
vrg_name has to be added to the TimeoutSampler at the beginning of `wait_for_replication_resources_creation` funtion as `check_vrg_existence` is being called over there.

Reply:
here i have improved checking vrg based on vrg name instead of getting all and check first index